File: python/raindrops/raindrops.py
def convert(number):
    output = ""

    if (number % 3 == 0):
        output += "Pling"
    if (number % 5 == 0):
        output += "Plang"
    if (number % 7 == 0):
        output += "Plong"

    if (output == ""):
        return str(number)

    return output

File: ruby/raindrops/raindrops.rb
class Raindrops
  def self.convert(int)
    result = ''
    result += 'Pling' if int % 3 == 0
    result += 'Plang' if int % 5 == 0
    result += 'Plong' if int % 7 == 0

    return int.to_s if result.empty?

    result
  end
end
